students/Chris_Kenyon/Lesson_3/Mailroom_Pt1.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+# Lesson_3 Activity 4 Mailroom Part 1
+
+
+def page_break():
+    """ Print a separator to distinguish new 'pages'"""
+    print("_"*75+"\n")
+
+
+def menu_page():
+    """ Main Menu Page """
+    while True:
+        try:
+            print("Please choose one of the following options(1,2,3):"
+                  "\n1. Send a Thank you. \n2. Create a report\n3. Quit")
+            option = int(input('--->'))
+        except ValueError:
+            print("you have made an invalid choice, try again.")
+            page_break()
+            continue
+        if option < 1 or option > 3:
+            print("you have made an invalid choice, try again.")
+            page_break()
+            continue
+        return option
+
+
+def send_thanks():
+    """ Send Thanks """
+    page_break()
+    while True:
+        list_names = [item[0] for item in donor_chart]
+        amount = []
+        try:
+            print("To whom would you like to say thank you?\n"
+                  "(type \"list\" for a full list of names or"
+                  "\"exit\" to return to the menu)")
+            name = input("--->")
+        except ValueError:
+            print("you have made an invalid choice, try again.")
+            page_break()
+            continue
+        if name == 'list':
+            print(("{}\n"*len(list_names)).format(*list_names))
+            continue
+        elif name in list_names:
+            amount = getAmount()
+            print(amount)
+            if amount == 'exit':
+                break
+            else:
+                donor_chart[list_names.index(name)][1].append(amount)
+        elif name.lower() == 'exit':
+            break
+        else:
+            addname = input("The name you selected is not in the list,"
+                            "would you like to add it(y/n)? ")
+            if addname[0].lower() == 'y':
+                amount = getAmount()
+                print(amount)
+                if amount == 'exit':
```

Comment:
Could this check for 'exit' be moved into the `getAmount()` function, to eliminate repetition and nested `if` statement here?

Reply:
is there a way to return a break when assigning a variable with a function
like 
While True
    amount = get_amount()
what can get_amount() return to break the while loop